src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,19 +25,25 @@
   private String user;
   private String text;
   private long timestamp;
+  private float sentimentScore;
 
   /**
-   * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
-   * random ID and uses the current system time for the creation time.
+   * Constructs a new {@link Message} posted by {@code user} with {@code text} content. 
+   * Generates a random ID and uses the current system time for the creation time.
    */
   public Message(String user, String text) {
```

Comment:
Is this constructor called from anywhere? If not you can get rid of it. If so, maybe update its documentation to mention it uses 0 as a sentiment score?

Reply:
I left that in just in case, to preserve compatibility with things Amanda and Gloria might be doing or with older code. I think it can probably actually be removed, unless somewhere is creating Message instances not using the form. I'll take a look and see if anything jumps out at me as a reason to leave it.

Reply:
I just searched `/src` for uses of `new Message` and it only found one in Datastore that accounts for the sentiment score change, and one in MessageServlet which also accounts for it because that's where it's being calculated right before creation. So I think it won't break anything to remove the old constructor.

Reply:
Yep sounds good. Even if it breaks stuff that Gloria or Amanda are working on, that's okay- working in the same code base often involves updating your code because somebody else made a change. We'll call this a good practice opportunity. 🐱